mobile/ios/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_static_framework")
+
+ios_static_framework(
+    name="ios_bindings",
```

Comment:
I think you need to run `buildifier` here - please make sure that you've got the hooks setup